src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --nav-active: #00acdc;
+  --header-color: #fff;
+  --nav-color: #000;
+  --nav-backgroundcolor: #fff;
+}
+
+* {
+  font-family:
```

Comment:
no need to set so many font families - just set one from the design and fallback family group (sans-serif)